cli/flox/src/commands/init/node.rs

```diff
@@ -377,24 +595,45 @@ impl Node {
         Ok(choice == 0)
     }
 
-    fn prompt_for_hook(&mut self, flox: &Flox) -> Result<bool> {
-        let (nodejs_detected, nodejs_version, _) = self.nodejs_message_and_version(flox)?;
+    /// Prompt whether to install npm or yarn when either is viable
+    fn prompt_for_package_manager(&mut self) -> Result<bool> {
+        let (found_npm, found_yarn, found_node) = match &self.package_manager {
+            Some(PackageManager::Both(found_npm, found_yarn, found_node)) => {
+                (found_npm.clone(), found_yarn.clone(), found_node.clone())
+            },
+            _ => unreachable!(),
```

Comment:
i guess i can tell why this is _supposed to be_ unreachable, but a comment would probably help too, or define `PackageManager::Both` a struct and pass that explicitly ( a comment is enough for now, functionally ):

 
```
enum PackageManager {
  Both(Both),
  ..
}

struct Both { .. }

fn prompt_for_package_manager(Both) -> Result<PackageManager>
```

Reply:
Fixed